export.py

```diff
@@ -2740,11 +2740,12 @@ def export_display(ri, rpass, scene):
         ri.DisplayChannel(source_type + ' %s' % (channel_name), params)
 
 
+
```

Comment:
I would still set rpass.output_files = [] here in case it's left over from last frame.